src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -970,6 +973,11 @@ def customizePerJob(self, job):
         if job.get('inputDatasetLocations'):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
+        # HighIO jobs
+        highIOjob = job.get('highIOjob')
+        if highIOjob:
+            jdl.append('+Requestioslots = 1\n')
```

Comment:
I tested that case with a single site. Works fine, jobs run as if the flag isn't set (ie. the pilot ignores the flag if it doesn't have highIO job slots).